### PR TITLE
Bert Question Answering - Azureml Notebook

### PR DESCRIPTION
**Pull request by sfatimar**
Pre Training Quantization with Optimum-Intel and Inference with OnnxRuntime with OpenVINO EP

### CODE REVIEW

**Comment by sfatimar** [comment]
@jywu-msft

**Comment by jywu-msft** [comment]
+@adrianlizarraga to also take a look since he looked at the dockerfile based scripts/demo.

**Comment by sfatimar** [comment]
@adrianlizarraga Please let us know your feedback on this pull request

File: python/OpenVINO_EP/azureml/quantization-aware-training-bert-squad-qa.ipynb [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "- # Environment Definition\n",
```
Comment (adrianlizarraga) [inline python/OpenVINO_EP/azureml/quantization-aware-training-bert-squad-qa.ipynb:246]:
```suggestion
        "# Environment Definition\n",
```

File: python/OpenVINO_EP/azureml/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Introduction
+ In these notebooks we are demonstrating quantization aware training using Neural Networks Compression Framework and run inference on quantized model using OpenVINO Execution provider.
```
Comment (adrianlizarraga) [inline python/OpenVINO_EP/azureml/README.md:2]:
```suggestion
In these notebooks we are demonstrating quantization aware training using Neural Networks Compression Framework and run inference on quantized model using OpenVINO Execution provider.
```

File: python/OpenVINO_EP/azureml/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* the Azure Machine Learning SDK
+* the Azure CLI and the Azure Machine learning CLI extension (> version 2.2.2)
+
+
+## Quantization Aware Training using Neural Networks Compression Framework 
+ - The training notebook is to demonstrate quantization aware training using Neural Networks Compression Framework [NNCF](https://github.com/AlexKoff88/nncf_pytorch/tree/ak/qdq_per_channel) through  
+ OpenVINO™ Integration with [Optimum](https://github.com/huggingface/optimum-intel/tree/v1.5.2).  
+- The output of the training is the INT8 optimized model and will stored on a local/cloud storage.  
+- To run the Quantization Aware Training we need to pass certain arguments to the script like model name, dataset name etc.  
+For more details please refer to the training notebook.
```
Comment (adrianlizarraga) [inline python/OpenVINO_EP/azureml/README.md:23]:
```suggestion

For more details please refer to the training notebook.
```

File: python/OpenVINO_EP/azureml/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## Inference on quantized model using OpenVINO Execution provider
+- In the Inference notebook we are using above Finetuned model for inferencing using OpenVINO Execution provider.    
+- We have options to run inference on single and multiple inputs.  
+- In case of multiple input, we need to provide input csv file. The sample csv file is available [here](https://github.com/intel/nlp-training-and-inference-openvino/blob/bert_qa_azureml/question-answering-bert-qat/onnxovep_optimum_inference/data/input.csv).  
+This file will be read and inference will be performed and the corresponding outputs will be saved as outputs.csv.  
+- In case of single input, the variables context and question are used to facilitate this.  
+  These variables are passed as an argument to the inference script. If they are empty strings, then the default behaviour is to read the input csv file and run inference.  
+For more details please refer to the inference notebook.
```
Comment (adrianlizarraga) [inline python/OpenVINO_EP/azureml/README.md:32]:
```suggestion

For more details please refer to the inference notebook.
```

File: python/OpenVINO_EP/azureml/bert-squad-qa-inference-using-int8-model.ipynb [hunk 5]
```diff
@@ -0,0 +1,741 @@
...
+        "STORAGEKEY = ''\r\n",
+        "# Share name\r\n",
+        "SHARE = ''\r\n",
+        "\r\n",
+        "MOUNT_PATH = \"/mnt/MyAzureFileShare\"\r\n",
+        "modelname = \"bert-large-uncased-whole-word-masking-finetuned-squad\"\r\n",
+        "\r\n",
+        "# Path to the Finetuned INT8 ONNX Model directory.\r\n",
+        "# e.g. modelpath = f\"{MOUNT_PATH}/models\"\r\n",
+        "modelpath = f\"{MOUNT_PATH}/\"\r\n"
```
Comment (adrianlizarraga) [inline python/OpenVINO_EP/azureml/bert-squad-qa-inference-using-int8-model.ipynb:443]: Is this where one would enter the path to the int8 model generated in the training notebook?

**Comment by sfatimar** [comment]
Is this where one would enter the path to the int8 model generated in the training notebook?
Yes, this is the place where we are mentioning path to the int8 model.

**Comment by sfatimar** [comment]
@adrianlizarraga All comments are taken care of

**Review by adrianlizarraga — APPROVED** [review]

**Comment by sfatimar** [comment]
@jywu-msft Please merge this latest Azure Notebook

**Review by jywu-msft — APPROVED** [review]

**Pull request merged**